aksel.nav.no/website/app/_ui/draft-overlay/DraftOverlay.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import {
+  useDraftModeEnvironment,
+  useIsPresentationTool,
+} from "next-sanity/hooks";
+import { VisualEditing } from "next-sanity/visual-editing";
+import { usePathname, useRouter } from "next/navigation";
+import { useState, useTransition } from "react";
+import { Button, Switch } from "@navikt/ds-react";
+import styles from "./DraftOverlay.module.css";
+import { disableDraftModeAction } from "./actions";
+
+function DraftOverlay() {
+  const router = useRouter();
+  const [pending, startTransition] = useTransition();
+  const environment = useDraftModeEnvironment();
+  const pathname = usePathname();
+  const isPresentation = useIsPresentationTool();
+  const isIFrame = typeof window !== "undefined" && window.self !== window.top;
+
+  const [enableVisualEditing, setEnableVisualEditing] = useState(false);
+
```

Comment:
We should probably always enable visual editing when in Presentation Tool.

```suggestion

  if (isPresentation) {
    return <VisualEditing />;
  }

```